docs/source/docs/explanation/formats/arrow.md

```diff
@@ -0,0 +1,194 @@
+
+- 0: `None`
+- 2: `Image`
+- 6: `PointCloud`
+
+### media.path (`string`)
+The path of the media. It could be a real path or a relative path, or `/NOT/A/REAL/PATH` if path is invalid.
+
+### media.bytes (`binary`)
+The binary data of the media. It could be `None` if one chooses not to save media when export.
+
+### media.attributes (`binary`)
+The attribute of the entity. The contents of it depends on `media.type`.
+The byte order is little-endian.
+
+**Image**
+
+![image attributes](../../../../images/arrow/image_attributes.png)
+
+**PointCloud**
+
+![pointcloud attributes](../../../../images/arrow/pointcloud_attributes.png)
+
+### annotations (`binary`)
+The annotations of the entity. The byte order is little-endian.
+The annotations are more than one like following.
+
+![annotations](../../../../images/arrow/annotations.png)
+
+**Supported annotation types:**
+
+- `Label`
+
+  ![label](../../../../images/arrow/label.png)
+- `Mask`
+
+  ![mask](../../../../images/arrow/mask.png)
+- `PolyLine`
+
+  ![polyline](../../../../images/arrow/shape.png)
+- `Polygon`
+
+  ![polygon](../../../../images/arrow/shape.png)
+- `Bbox`
+
+  ![bbox](../../../../images/arrow/shape.png)
+- `Points`
+
+  ![point](../../../../images/arrow/point.png)
+- `Caption`
+
+  ![caption](../../../../images/arrow/caption.png)
+- `Cuboid3d`
+
+  ![cuboid3d](../../../../images/arrow/cuboid3d.png)
+- `Ellipse`
+
+  ![ellipse](../../../../images/arrow/shape.png)
+
+### attributes (`binary`)
+The attributes of the entity. The byte order is little-endian.
+
+![attributes](../../../../images/arrow/attributes.png)
+
+## Import Arrow dataset
+
+A Datumaro project with a Arrow source can be created in the following way:
+
+```console
+datum create
+datum import --format arrow <path/to/dataset>
+```
+
+It is possible to specify project name and project directory. Run
+`datum create --help` for more information.
+
+An Arrow dataset directory should have the following structure:
+
+<!--lint disable fenced-code-flag-->
+
+```
+└─ Dataset/
+    ├── <subset_name_1>-0-of-2.arrow
+    ├── <subset_name_1>-1-of-2.arrow
+    ├── <subset_name_2>-0-of-1.arrow
+    └── ...
+```
+
+If your dataset is not following the above directory structure,
+it cannot detect and import your dataset as the Arrow format properly.
+
+To make sure that the selected dataset has been added to the project, you can
+run `datum project info`, which will display the project information.
+
+## Export to other formats
+
+It can convert Datumaro dataset into any other format [Datumaro supports](/docs/data-formats/supported_formats/).
+To get the expected result, convert the dataset to formats
+that support the specified task (e.g. for panoptic segmentation - VOC, CamVID)
+
+There are several ways to convert a Datumaro dataset to other dataset formats
+using CLI:
+
+- Export a dataset from Datumaro format to VOC format:
+
+```console
+datum create
+datum import -f arrow <path/to/dataset>
+datum export -f voc -o <output/dir>
```

Comment:
```suggestion
datum project create
datum project import -f arrow <path/to/dataset>
datum project export -f voc -o <output/dir>
```

Reply:
I copied them from datumaro_binary.md. Does it also need to be changed? I will, if so

Reply:
We need to do it for all formats in a separate PR.